src/pages/Reservations.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import Cars from '../cars.json';
+
+import '../assets/App.css';
+import '../assets/reservations.css';
+
+const Reservations = () => (
+  <div className="content">
+    <h2>Reservations page</h2>
+    <div className="reserved-cars">
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[0].image} alt={Cars[0].id} />
+        </div>
+        <p>{Cars[0].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[1].image} alt={Cars[1].id} />
+        </div>
+        <p>{Cars[1].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[2].image} alt={Cars[2].id} />
+        </div>
+        <p>{Cars[2].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[0].image} alt={Cars[0].id} />
+        </div>
+        <p>{Cars[0].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[1].image} alt={Cars[1].id} />
+        </div>
+        <p>{Cars[1].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[2].image} alt={Cars[2].id} />
+        </div>
+        <p>{Cars[2].model}</p>
+        <p>City</p>
+        <p>Date</p>
+      </div>
```

Comment:
Instead of hard coding, please kindly use the map function and make this dynamic because when you get the data from the backend the list will include the unpredictable amount of cars and you are gonna need to fix this in the future anyway. So fixing this right now will make you free from solving one problem in the future.

Reply:
## done ✅ 
> I ve hard code it because this will be only a list of reserved car but I can modify it later when I ll get real data, for now I implemented it as you suggest 👍🏾